src/protagonist/DLCS.Repository/DeliveryChannels/DefaultDeliveryChannelRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using DLCS.Core.Caching;
+using DLCS.Model.DeliveryChannels;
+using LazyCache;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace DLCS.Repository.DeliveryChannels;
+
+public class DefaultDeliveryChannelRepository : IDefaultDeliveryChannelRepository
+{
+    private readonly IAppCache appCache;
+    private readonly CacheSettings cacheSettings;
+    private readonly ILogger<DefaultDeliveryChannelRepository> logger;
+    private readonly DlcsContext dlcsContext;
+
+    public DefaultDeliveryChannelRepository(
+        IAppCache appCache,
+        ILogger<DefaultDeliveryChannelRepository> logger,
+        IOptions<CacheSettings> cacheOptions,
+        DlcsContext dlcsContext)
+    {
+        this.appCache = appCache;
+        this.logger = logger;
+        cacheSettings = cacheOptions.Value;
+        this.dlcsContext = dlcsContext;
+    }
+    
+    public List<DefaultDeliveryChannel> GetDefaultDeliveryChannelsForCustomer(int customerId, int space)
```

Comment:
Can keep the same method signature but I think it'll be worth caching deliveryChannels per customer. Rather than customer + space. In the vast majority of instances each space will just have space 0 defaults so saves a lot of duplication in memory.
One customer we have has > 700 spaces so this current method could result in a _lot_ of unnecessary cached data.